bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -2063,12 +2067,18 @@ std::error_code DataAggregator::parseMMapEvents() {
                << " using file offset 0x" << Twine::utohexstr(MMapInfo.Offset)
                << ". Ignoring profile data for this mapping\n";
         continue;
-      } else {
-        MMapInfo.BaseAddress = *BaseAddress;
       }
+      MMapInfo.BaseAddress = *BaseAddress;
```

Comment:
Do you think we need a check here that `MMapInfo.BaseAddress` is calculated consistently across multiple segment mappings?